langs/befunge/befunge

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh -e
+
+[ "$1" = "-V" ] && exec cfunge -V
+
+cd /tmp
+
+cat - > code
```

Comment:
Automatic file extensions are a free service from the interpreter. Most importantly, for the first time, I've set the execute bit on the non-containerized wrapper shell.